lib/reporting/drop_off_report.rb

```diff
@@ -427,6 +404,14 @@ def data
 
         fetch_results.each do |row|
           event_users[row['name']] << row['user_id']
+
+          event = row['name']
+          case event
+          # rubocop:disable Layout/LineLength
+          when Events::IDV_FINAL_RESOLUTION
+            event_users[Results::IDV_FINAL_RESOLUTION_VERIFIED] << user_id if row['identity_verified'] == '1'
+          end
+          # rubocop:enable Layout/LineLength
```

Comment:
I'd suggest multi-lining the condition rather than disabling lint.

```suggestion
          when Events::IDV_FINAL_RESOLUTION
            if row['identity_verified'] == '1'
              event_users[Results::IDV_FINAL_RESOLUTION_VERIFIED] << user_id
            end
          end
```

Reply:
Sure, I was just sticking with the convention used in [identity_verification_report.rb](https://github.com/18F/identity-idp/blob/d0d68ffbdb1321d246f8377af1f5c9026edf93c2/lib/reporting/identity_verification_report.rb#L231)